packages/components/timepicker/package.json

```diff
@@ -37,6 +37,7 @@
   "dependencies": {
     "@babel/runtime": "^7.6.2",
     "@contentful/forma-36-react-components": "^3.83.0",
+    "@types/react": "16.14.2",
```

Comment:
this one shouldn't be here, because `react` is a peer dependency of the package and will people will be using different react versions

Reply:
I'm not sure I agree since DefinitelyTyped doesn't follow semantic versioning. We can't be sure that the user's version works with our types.

Reply:
It'd also require non-TS users to add `@types/react` to their `package.json` to avoid NPM complaining.

Reply:
Not-TS users shouldn't be affected

Reply:
But they'd still have to install the package to avoid NPM warnings on install, no?